restcomm/restcomm.dao/src/main/java/org/restcomm/connect/dao/ExtensionsConfigurationDao.java

```diff
@@ -97,4 +97,16 @@
      * @return
      */
     boolean validate(ExtensionConfiguration extensionConfiguration);
+
+    /**/
+    ExtensionConfiguration getAccountExtensionConfiguration(String accountSid, String extensionSid);
```

Comment:
@abdulazizali77 please add short description as javadoc comment to each one of the new methods here.